tests/console/install_single_package.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple test for installing any given package
+# Maintainer: soulofdestiny <mgriessmeier@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    select_console 'root-console';
+
+    pkcon_quit;
+
+    # add specific repository which contains the package
+    zypper_call("ar -f " . get_required_var('PACKAGE_REPO') . " testrepo");
```

Comment:
I suggest to add a `… if get_var('PACKAGE_REPO')` so that we can also test a single package which is already covered by the enabled repositories.